holoviews/plotting/plotly/renderer.py

```diff
@@ -68,3 +112,18 @@ def load_nb(cls, inline=True):
         """
         import panel.models.plotly # noqa
         cls._loaded = True
+
+
+
+    @classmethod
+    def trigger_plot_pane(cls, plot_id, fig_dict):
```

Comment:
In the latest version panel adds itself to `Plot._panel`, I think we just have to propagate this down to all child plots to avoid the ``_plot_panes`` registry entirely.